### PR TITLE
added pagination

### PR DESCRIPTION
**Pull request by json-bell**

### CODE REVIEW

**Comment by json-bell** [comment]
The loading message isn't working right now too, not sure why - commenting out the finally brings it back, but I never see the red of loading

File: src/components/ArticleList.jsx [hunk 1]
```diff
@@ -2,35 +2,42 @@ import { useEffect, useState } from "react";
...
     apiClient
-      .get("articles")
+      .get("articles", { params: { ...listPagination } })
       .then(({ data: { articles, total_count } }) => {
         setArticles(articles);
         setTotalCount(total_count);
-        console.log(articles[0]);
       })
       .catch(console.log)
       .finally(setIsArticlesLoading(false));
```
Comment (nc-christian-dutton) [inline src/components/ArticleList.jsx:22]: Move this out of a .finally and into the .then, after you set the articles. It should work then for the loading pattern! I'm not 100% sure on the order of a .finally with .then and catch - but it doesn't seem to play ball with the asynchronicity!

**Comment by nc-christian-dutton** [comment]
Looking really good so far, styling is really nice and responsive. Great work with the pagination. 

I would try and format those dates - you can do it yourself or with a library. 
Given a suggestion above for sorting the loading pattern - maybe you can extract this out into it's own Loading component too! 
I would extract out the api call to the calls file as well!

File: src/components/ArticleList.jsx (hunk 1 above)
Comment (json-bell) [reply src/components/ArticleList.jsx:22]: Thanks!

**Pull request merged**